app/helpers/measures_helper.rb

```diff
@@ -1,4 +1,40 @@
 module MeasuresHelper
+  DEFAULT_MODAL_CONFIG = {
+    content_file: nil,
+    overwrite: nil,
+  }.freeze
+
+  def modal_partial_options_for(declarable, measure)
+    config = measure_condition_config_for(
+      declarable.goods_nomenclature_item_id,
+      measure.additional_code.code,
+      measure.measure_type.id,
+    )
+
+    overwrite = config['overwrite']
+    content_file = config['content_file']
+
+    if content_file.present?
+      if overwrite
+        { partial: "measures/measure_condition_replacement_modals/#{content_file}" }
+      else
+        { partial: 'measures/measure_condition_modal_augmented', locals: { modal_content_file: "measures/measure_condition_replacement_modals/#{content_file}", measure: measure } }
+      end
+    else
+      { partial: 'measures/measure_condition_modal_default', locals: { measure: measure } }
+    end
+  end
+
+  def measure_condition_config_for(goods_nomenclature_item_id, additional_code, measure_type_id)
+    config = TradeTariffFrontend.measure_condition_modal_config.find do |modal_config|
+      modal_config['goods_nomenclature_item_id'] == goods_nomenclature_item_id &&
+        modal_config['additional_code'] == additional_code &&
+        modal_config['measure_type_id'] == measure_type_id
+    end
+
+    config || DEFAULT_MODAL_CONFIG
+  end
+
```

Comment:
During my sleep (really!) I realized that we should - if we have a couple - of hours, move these responsibilities into a class: 
`MeasureConditionAugmenter`, `MeasureConditionReplacer` 
that would simplify the naming of the public methods,
and we could place the config loader into the initializer.